controllers/skipjob/service_account.go

```diff
@@ -0,0 +1,31 @@
+package skipjobcontroller
+
+import (
+	"context"
+	"github.com/kartverket/skiperator/pkg/util"
+	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	skiperatorv1alpha1 "github.com/kartverket/skiperator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *SKIPJobReconciler) reconcileServiceAccount(ctx context.Context, skipJob *skiperatorv1alpha1.SKIPJob) (reconcile.Result, error) {
+
+	serviceAccount := corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Namespace: skipJob.Namespace, Name: util.ResourceNameWithHash(skipJob.Name, skipJob.Kind)}}
```

Comment:
As it is used multiple places, could the call to `util.ResourceNameWithHash` be a function on a `SKIPJob` instance?

Reply:
Not sure I follow, could you elaborate? @evenh

Reply:
It would read nicer to have `skipJob.HashedName()` instead of `util.ResourceNameWithHash(skipJob.Name, skipJob.Kind)` 5 different places. But it is indeed a nitpick.

Reply:
Somewhat annoying due to the fact that you can't call util functions in `skipjob_types.go` due to circular dependencies. That might however mean we should extrude some of the util functions to non-util functions, if they're package specific

Reply:
Nevermind, seems that was fixed somehow 🤔

Reply:
I'll see what I can do. I believe its the controller status field of Applications that cause the circular dependency. Should move this to ApplicationReconciler instead